### DISCUSSION TITLE
Sentinelhub use case

### DISCUSSION THREAD

**Discussion by moshiour** [opengeos/segment-geospatial#142] [General]
Hi,

It would be really nice, If you could provide a example of to apply sentinelhub data. 

Thank you in advance!

**Comment by giswqs** [comment]
samgeo accepts GeoTIFF format. You can download data from SentinelHub.

**Reply by moshiour** [reply]
Hi,
Thanks for reply. I have done exactly the same as downloaded the GeoTIFF image from sentinelhub. Apply the samgeo to produce mask as follows :
1. sam.generate(image, mask, batch=True, foreground=True, unique=True), Results: 100%|██████████| 24/24 [02:12<00:00,  5.50s/it] 
 2. sam.show_masks(cmap="binary_r"); Results: produce binary map
 3. sam.show_anns(axis="off", alpha=1, output="annotation.tiff"); Results: Please run generate() first.
 
So, it doesn't show the annotation.  Please let me know if I am doing something wrong.

Thank you in advance!
